### ISSUE TITLE
What's new: switch to a link to play store What's new

### ISSUE THREAD

**Issue by garvankeeley** [mozilla/MozStumbler#1489]
Too difficult to keep this in sync with the translations with the current method: if we release version x.y and a translation for the what's new is added, we need to release version x.(y+1) to get the updated translation.

**Comment by crankycoder** [comment]
As per IRC conversation, let's just push this to a text blurb on the play store.

We can just provide a link in the changelog dialog.

**Comment by Djfe** [comment]
that's the way Chrome Beta does it, but I don't really like the approach (barely anyone would open up his webbrowser to get that info)

either a changelog in the app (where we could do translations)

or a list new changes that is not finished could be translated continuosly until release
if the translator didn't finish his translation before the release the translation won't be there but at least later in the app itself

**Comment by garvankeeley** [comment]
I agree that people don't open the link to get the info very often. But the procedural overhead of keeping that list up-to-date means it is likely to get overlooked/outdated.

**Comment by crankycoder** [comment]
I'm thinking that for official release builds to the Google Playstore, we can update that - otherwise we should just link to github's release page.

**Comment by Djfe** [comment]
+1
Github get's releases earlier and not many features will be added to it afterwards
-> translators have some time to translate the changelog

**Comment by lua86** [comment]
I don't know much about android, but...
How about an iframe? I mean, open the text from a file in github? Open it within the app, like the leaderboard.

**Comment by Djfe** [comment]
No I-Frame pls
We can just load the text from a text file on GitHub and output the text. (but no webview pls)
